spec/api-power-save-blocker-spec.js

```diff
@@ -1,13 +1,17 @@
 const {powerSaveBlocker} = require('electron').remote
-const assert = require('assert')
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+
+const {expect} = chai
+chai.use(dirtyChai)
 
 describe('powerSaveBlocker module', () => {
   it('can be started and stopped', () => {
-    assert.equal(powerSaveBlocker.isStarted(-1), false)
+    expect(powerSaveBlocker.isStarted(-1)).to.be.false()
     const id = powerSaveBlocker.start('prevent-app-suspension')
-    assert.ok(id != null)
-    assert.equal(powerSaveBlocker.isStarted(id), true)
+    expect(id).to.not.be.null()
```

Comment:
If it's not `null`, what it is? A number, an object, a symbol?

Reply:
fixed